metrics.go

```diff
@@ -230,11 +230,13 @@ func (m *metrics) count(name string, enabled bool) {
 	m.metricsChannels.count <- metric{Name: name, Enabled: enabled}
 }
 
-func (m *metrics) countVariants(name string, variantName string) {
+func (m *metrics) countVariants(name string, enabled bool, variantName string) {
 	if m.options.disableMetrics {
 		return
 	}
 
+	m.add(name, enabled, 1)
+	m.metricsChannels.count <- metric{Name: name, Enabled: enabled}
```

Comment:
I believe this is necessary but I'm not 100% sure - since we're now incrementing the counter for feature toggles, it makes sense to now add this to the channel

Same as https://github.com/Unleash/unleash-client-go/pull/123#discussion_r1045534318